axelor-human-resource/src/main/java/com/axelor/apps/hr/service/extra/hours/ExtraHoursService.java

```diff
@@ -53,4 +55,6 @@ public Message sendValidationEmail(ExtraHours extraHours)
   public Message sendRefusalEmail(ExtraHours extraHours)
       throws AxelorException, ClassNotFoundException, InstantiationException,
           IllegalAccessException, MessagingException, IOException;
+
+  public BigDecimal compute(ExtraHours extraHours);
 }
```

Comment:
no needs to return anything , you can just set totalQty  of extraHours and get it on controller